### ISSUE TITLE
Mod hangs Quilt server on exit.

### ISSUE THREAD

**Issue by Magicrafter13** [plasmoapp/plasmo-voice#340]
Despite all signs (in the log) showing the server has finished its tasks, java remains open and will not close without the user pressing ^C, or using `kill`.

Latest Quilt server as of Monday. Mod version 2.0.5.

Not really sure what other info to provide, but I'll edit this as necessary.

**Comment by otomir23** [comment]
Same, but on fabric. Here's the thread dump:
```
"su.plo.voice.libs.kotlinx.coroutines.DefaultExecutor" #107 daemon prio=5 os_prio=0 cpu=0.00ms elapsed=300.44s tid=0x000001fc1c7abf30 nid=0x54e4 waiting on condition  [0x0000005f39ffe000]
   java.lang.Thread.State: TIMED_WAITING (parking)
        at jdk.internal.misc.Unsafe.park(java.base@17.0.7/Native Method)
        - parking to wait for  <0x00000000d2693808> (a su.plo.voice.libs.kotlinx.coroutines.DefaultExecutor)
        at java.util.concurrent.locks.LockSupport.parkNanos(java.base@17.0.7/LockSupport.java:252)
        at su.plo.voice.libs.kotlinx.coroutines.DefaultExecutor.run(DefaultExecutor.kt:121)
        at java.lang.Thread.run(java.base@17.0.7/Thread.java:833)
```
Server stops prefectly fine without Plasmo Voice.

**Comment by Magicrafter13** [comment]
Well that's better than what I got - which is nothing, my log showed nothing of note at the end. Hopefully that tiny output can help in some way...

**Comment by otomir23** [comment]
I did not get this output, I manually run `jstack` to get list of all Java threads. That was the only way that I could detect which mod was causing this without removing them one-by-one.

**Comment by Absolucy** [comment]
Also getting this on Forge with version 2.0.5 on Minecraft 1.19.2 / Forge 43.2.14 (All The Mods 8)

**Comment by apehum** [comment]
Should be fixed in 2.0.7: https://github.com/plasmoapp/plasmo-voice/releases/tag/2.0.7
